mailet/standard/src/main/java/org/apache/james/transport/mailets/AddFooter.java

```diff
@@ -53,7 +53,7 @@
     
     @Override
     public void init() throws MessagingException {
-        plainTextFooter = (String) getInitParameter(MailetConfiguration.TEXT.name());
+        plainTextFooter = getInitParameter("text", String.class);
```

Comment:
Why don't we 'stick' with the previous String version, that is directly the type that we want?

Reply:
You mean the explicit type casting version? I think it's redundant here because using generics can help us to do the auto casting job.